source/atlas-cli-deploy-docker.txt

```diff
@@ -167,6 +171,138 @@ deployment with `Docker Compose <https://docs.docker.com/compose/>`__.
 
          mongosh mongodb://root:root@localhost:27778/?directConnection=true
 
+Persist Data Across Runs with Docker Compose
+--------------------------------------------
+
+You can persist data across multiple runs with Docker Compose. 
+Persisting data helps to ensure that data isn't lost between runs. Data 
+remains available across runs of Docker Compose.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Install and start Docker.
+
+      To learn more, see the `Docker documentation <https://docs.docker.com/desktop/install/mac-install/>`__.
+
+   .. step:: Install Docker Compose.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         brew install docker-compose
+
+   .. step:: Create a script to start and stop your local deployment.
+
+      a. Copy and paste the following script and name the file
```

Comment:
Just confirming, it doesn't matter where they save this on their machine, right?

Reply:
it does matter. It has to be in the same location as `docker-compose.yml`.